config/default/manager_auth_proxy_patch.yaml

```diff
@@ -10,7 +10,7 @@ spec:
     spec:
       containers:
       - name: kube-rbac-proxy
-        image: gcr.io/kubebuilder/kube-rbac-proxy:v0.10.0
+        image: gcr.io/kubebuilder/kube-rbac-proxy:v0.8.0
```

Comment:
Why was this downgraded?

Reply:
gcr.io/kubebuilder/kube-rbac-proxy:v0.10.0 disappears.